index.html

```diff
@@ -1,27 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta
+  <head>
+    <meta charset="UTF-8" />

```

Comment:
I see you added /> to the meta and link tags. There is no need in HTML, unless you are using XHTML :)

Reply:
Hi! To be completely honest, I did not know that I did any changes in the index file... This PR was only suppose to be having changes in the news page, so my fault! Do you have an option to like "ignore" the changes made in the index.html or is this something I have to go back and change in the branch I created?